mapping/IDEAv2.2toFEDEFLv1.0.3.csv

```diff
@@ -0,0 +1,1022 @@
+IDEAv2.2,hydrogen fluoride,97891001-eb01-4900-befb-162ab4e14401,Emissions/ground/unspecified,kg,=,1,Hydrofluoric acid,bc5fb7f9-280e-3888-9d3c-2cc000809dce,emission/ground,kg,Hottle,Edelen,8/24/2020
+IDEAv2.2,miscellaneous plantation,46c693e4-57a5-405e-9b88-43f15ec2ad86,Resources/land use/land occupation,m2year,=,1,Land use,46b31bbe-1c77-3a50-a31d-5378c1e47dd6,resource/ground/human-dominated/agricultural,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,orchard,47f398fe-84e3-421f-9724-97fe3482f048,Resources/land use/land occupation,m2year,=,1,Land use,46b31bbe-1c77-3a50-a31d-5378c1e47dd6,resource/ground/human-dominated/agricultural,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,rice paddy,666804c4-eaed-4991-99e0-fd47c91c829c,Resources/land use/land occupation,m2year,=,1,Land use,46b31bbe-1c77-3a50-a31d-5378c1e47dd6,resource/ground/human-dominated/agricultural,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,"arable, except rice paddy",f160f933-4bd6-4b63-adb4-03c21e37b3cf,Resources/land use/land occupation,m2year,=,1,Land use,46b31bbe-1c77-3a50-a31d-5378c1e47dd6,resource/ground/human-dominated/agricultural,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,other land use type,f7e8470a-642f-4c91-ae72-de8f3aa365e0,Resources/land use/land occupation,m2year,=,1,Land use,6438548e-77ef-35c2-97d9-8471835bcd8f,resource/ground,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,transport artery,354d6b1e-b5f9-4b67-a651-fac14f841796,Resources/land use/land occupation,m2year,=,1,Land use,cc39b220-d9ed-30a0-a6b8-90fd40b502e7,resource/ground/human-dominated/commercial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,building site,4243aedb-0531-4e7c-9074-29d7e4e19204,Resources/land use/land occupation,m2year,=,1,Land use,cc39b220-d9ed-30a0-a6b8-90fd40b502e7,resource/ground/human-dominated/commercial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,barren land,60a31ee1-b87b-40e9-8afb-b5ad60994c06,Resources/land use/land occupation,m2year,=,1,Land use,879bee81-f5fc-351e-9d0c-9cd219d40cd6,resource/ground/terrestrial/barren land,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,forest,fcd5e16b-e4c5-4c60-b045-042d30abd42d,Resources/land use/land occupation,m2year,=,1,Land use,1147e66c-db3e-303c-b7c7-89b006d98881,resource/ground/terrestrial/forest,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,mineral extraction site,CC0F5788-8059-4253-8498-D7B8D3CD3CD6,Resources/land use/land occupation,m2year,=,1,Land use,4b92c75e-e528-3f23-bc67-e92977682eb4,resource/ground/human-dominated/industrial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,dump yard,dd8c94cc-b5c2-4736-9dfc-668bce100891,Resources/land use/land occupation,m2year,=,1,Land use,4b92c75e-e528-3f23-bc67-e92977682eb4,resource/ground/human-dominated/industrial,m2*a,Hottle,Edelen,8/24/2020
+IDEAv2.2,Iron manganese ore,1404b845-05d0-4d16-94f5-999775446c55,Resources/ground/Non-renewable material resources,kg,>,0.5,Iron ore,6fca43a7-13fb-315d-a184-8af7d40ff1d3,resource/ground,kg,Hottle,,8/28/2020
```

Comment:
@kshobatake Please see mappings in this line forward where we split 1 IDEA flow across two FEDEFL flows. The conversion factors are also split in equal proportion to add to 1. If you have insight into better % composition data of the members of those groups in the IDEA flows, that could be used to improve the conversion factors